src/Test/QuickCheck.purs

```diff
@@ -100,7 +100,26 @@ instance arbString :: Arbitrary String where
 instance coarbString :: CoArbitrary String where
   coarbitrary s = coarbitrary $ (S.charCodeAt 0 <$> S.split "" s)
 
--- | A newtype for `String` whose `Arbitrary` instance generated random 
+instance arbUnit :: Arbitrary Unit where
+  arbitrary = return unit
+
+instance coarbUnit :: CoArbitrary Unit where
+  coarbitrary _ = perturbGen 1
+
+instance arbOrdering :: Arbitrary Ordering where
+  arbitrary = do
+    n <- (3 *) <$> uniform
```

Comment:
I looked through the stuff in `Gen` but there wasn't anything obviously suitable. Could you have a look too?